jsapp/js/components/modalForms/projectSettings.es6

```diff
@@ -869,7 +873,12 @@ class ProjectSettings extends React.Component {
 
           {this.props.context === PROJECT_SETTINGS_CONTEXTS.EXISTING && this.props.iframeUrl &&
             <bem.FormView__cell m='iframe'>
-              <iframe src={this.props.iframeUrl} />
+              <iframe
+                id='kc-settings-iframe'
+                src={this.props.iframeUrl}
+                scrolling='no'
+                style={{width: '1px', minWidth: '100%'}}
```

Comment:
why do you have this style?  Why isn't it in css?   I'm assuming it's needed for iframe-resizer to work.  Please add a comment that says that

Reply:
OK, I see the note in the package.

> Note: Using min-width to set the width of the iFrame, works around an issue in iOS that can prevent the iFrame from sizing correctly.

Can you add this note to the css file when you move the style there?

Reply:
fixed :)